src/main/java/me/ars/pokerbot/config/IrcBotConfig.java

```diff
@@ -0,0 +1,20 @@
+package me.ars.pokerbot.config;
+
+public class IrcBotConfig {
+    public class Irc {
+        public String server;
+        public Integer port;
+        public String serverPassword;
+        public String nick;
+        public String channel;
+        public String channelPassword;
+        public String adminPassword;
+        public Character commandPrefix;
+        public Boolean verbose = true;
```

Comment:
Would be nice to have verbosity off as default.

Reply:
Makes sense!